test/enginetest.go

```diff
@@ -0,0 +1,51 @@
+package test
+import (
+	//"SOMAS2023/internal/server"
+	physics "SOMAS2023/internal/common/physics"
+	utils "SOMAS2023/internal/common/utils"
+	"fmt"
+)
+
+func TestEngine(){
+	fmt.Println("Test for physical engine")
+	var preState utils.PhysicalState
+	preState.Position.X = 1
+	preState.Position.Y = 2
+	preState.Velocity = 0
+	preState.Acceleration = 0
+	preState.Mass = 1
+	var targetCoordinate utils.Coordinates
+	targetCoordinate.X = 5
+	targetCoordinate.Y = 15
+
+	var orientation float64
+	var l2distance float64
+	var thresdistance float64 = 0.5
+	var pedalforce float64 = 2
+	var newState utils.PhysicalState
+	var count int =0
+
+	orientation = physics.ComputeOrientation(preState.Position,targetCoordinate)
+	l2distance = physics.ComputeDistance(preState.Position,targetCoordinate)
+	fmt.Println("Orientation is ", 180*orientation)
```

Comment:
These should be assertions - there is no clear indication that the test actually failed unless the numbers aren't what you expect (and how would you know what to expect?)

Reply:
yeah, I need to make some change to fit the standard of testing